frameworks/helloworld/tests/test_sanity.py

```diff
@@ -19,21 +21,65 @@
 
 
 @pytest.fixture(scope="module", autouse=True)
-def configure_package(configure_security):
+def helloworld_service(configure_security):
     try:
+        service_options = {"service": {"name": foldered_name}}
         sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
         sdk_upgrade.test_upgrade(
             config.PACKAGE_NAME,
             foldered_name,
             config.DEFAULT_TASK_COUNT,
-            additional_options={"service": {"name": foldered_name}},
+            additional_options=service_options,
         )
 
-        yield  # let the test session execute
+        yield {"package_name": config.PACKAGE_NAME, **service_options}
     finally:
         sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
 
 
+@pytest.mark.sanity
+@pytest.mark.smoke
+def test_install():
+    config.check_running(sdk_utils.get_foldered_name(config.SERVICE_NAME))
+
+
+@pytest.mark.sanity
+@pytest.mark.metrics
+@pytest.mark.smoke
+@pytest.mark.dcos_min_version("1.9")
+def test_metrics_cli_for_scheduler_metrics(helloworld_service):
```

Comment:
Since `helloworld_service` has `autouse=true`, we don't need to add this fixture explicitly right?

Reply:
actually, i am not sure about the behavior of explicitly importing a module scoped fixture in each of the tests.

Reply:
I would also have to check exactly what the implications of this is, but I do prefer to make the dependency explicit.